packages/ai-jsx/package.json

```diff
@@ -29,6 +29,10 @@
       "import": "./dist/esm/core/completion.js",
       "require": "./dist/cjs/core/completion.js"
     },
+    "./core/image-gen": {
+      "import": "./dist/esm/core/image-gen.js",
+      "require": "./dist/cjs/core/image-gen.js"
+    },
```

Comment:
I think it would be good to add an example usage of this in the `examples` directory. And can we add an entry to the docs about this? Maybe on https://docs.ai-jsx.com/guides/prompting? Or https://docs.ai-jsx.com/getting-started?